guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,131 @@
+
+Depending on whether the tax rates are `included_in_price`, Solidus will store the sum of tax adjustments inthe line item's `included_tax_total` or `additional_tax_total` columns.
+
+The same procedure is executed on an orders shipments.  
+
+Finally, the `included_tax_total` and `additional_tax_total` of all line items and shipments are summed up, and stored in the order's `included_tax_total` and `additional_tax_total`. The `included_tax_total` column does not affect the order's total, while the `additional_tax_total` does.
+
+Every time an order is changed, the taxation system will check whether any tax adjustments need to be changed and update all the taxation-relevant totals.
+
+***
+To best appreciate how Solidus' taxation system works, have a look at the [taxation integration specs](https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/tax/taxation_integration_spec.rb).
+***
+
+
+## Pricing
+
+In VAT-style jurisdictions, taxes have to be included in the price shown to customers (in index and product display pages as well as when adding them to the cart). In order to configure this, you need to first configure your tax rates, categories, and zones, and then update your products and variants with the "Rebuild VAT prices" checkbox checked. Solidus will then proceed to calculate the correct net price and create prices for all countries with `included_in_price` VAT rates. It will also generate a fallback "export" price, where the prices `country_iso` is `nil` (not set). New products will behave as if that checkbox is checked. You can adjust the prices in the "Prices" tab if you need to.
+
+Solidus can handle several prices for the same country, and will always select the most recently updated price.
+
+## Prices in the backend
+
+In the Solidus admin backend, prices will be shown and entered with or without VAT, depending on the configuration value `Spree::Config.admin_vat_country_iso`. This has to be set to the two-letter ISO code of the country your admin thinks in. For a store that mostly sells to Germany, this would be set to `DE`. The effect of this is that now all prices in the backend will assume to have the German VAT rate included.
+
+## Prices in the frontend
+
+When a customer first browses your store, we do not know which jurisdiction she lives in. We will, therefore, have to make an assumption about her whereabouts. This assumption is the `cart_tax_country_iso` property on the `Spree::Store` model - meaning that you can assume your customers to be from different countries depending on which store she browses.
+
+This way, you can have, for example, two different stores for France and Germany (`my-shop.com/fr` and `my-shop.com/de`), where each shows different prices to accustom for the different VAT rates in either country.
```

Comment:
Yes

> where each shows different prices to accustom for the different VAT rates in either country

where each store shows different prices to account different VAT rates in either country